packages/manager/modules/user-account/src/user/advanced/user-advanced.controller.js

```diff
@@ -1,14 +1,13 @@
 export default /* @ngInject */ function(
-  userAccountServiceInfos,
+  UserAccountServiceInfos,
```

Comment:
why did you change userAccountServiceInfos to UserAccountServiceInfos?iss it intentional?

Reply:
Yes it is. The class name while exporting is `userAccountServiceInfos` and actual service itself is `UserAccountServiceInfos`